BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/AddEditSendItemView.swift

```diff
@@ -489,6 +524,37 @@ struct AddEditSendItemView: View { // swiftlint:disable:this type_body_length
             .pickerStyle(SegmentedPickerStyle())
         }
     }
+
+    // MARK: Private Functions
```

Comment:
🤔 Can all or some of these functions be moved to the `AddEditSendItemProcessor` given some additional `AddEditSendItemAction`?

Reply:
yeah, we can certainly do that.  Good idea!